docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -2204,6 +2204,34 @@ array_dims(array)
 
 - list_dims
 
+### `array_distinct`
```

Comment:
Could you also add links for `array_distinct` and `list_distinct` here?
https://github.com/apache/arrow-datafusion/blob/4cd3c433004a7a6825643d6b3911db720efe5f76/docs/source/user-guide/sql/scalar_functions.md#L1943-L1952

Reply:
Addressed